modules/@apostrophecms/rich-text-widget/ui/apos/components/AposTiptapLink.vue

```diff
@@ -138,7 +138,7 @@ export default {
     },
     active(newVal) {
       if (newVal) {
-        this.hasLinkOnOpen = !!(this.docFields.data.href);
```

Comment:
We clear the `href` prop during data population so this isn't a good property to check. Check the unmodified attr.